etna/pipeline/pipeline.py

```diff
@@ -53,8 +54,12 @@ def _forecast(self) -> TSDataset:
         if self.ts is None:
             raise ValueError("Something went wrong, ts is None!")
 
-        future = self.ts.make_future(self.horizon)
-        predictions = self.model.forecast(ts=future)
+        if isinstance(self.model, DeepBaseModel):
```

Comment:
I guess we need to come up with the separate pipeline for the networks in the future

Reply:
We can add now I think